src/signals/incident-management/containers/DefaultTextsAdmin/saga.test.js

```diff
@@ -195,7 +195,7 @@ describe('/signals/incident-management/containers/DefaultTextsAdmin/saga', () =>
         .next()
         .put(
           actions.showGlobalNotification({
-            title: `2 Standaard teksten opgeslagen voor ${subcategory.value}, ${status.value}`,
+            title: `Standaard teksten bijgewerkt voor ${subcategory.value}, ${status.value}`,
```

Comment:
Does this test add anything new compared to the test where 1 default text saved?

Reply:
Yes it does. One test tests for two states and that only the right state is selected, the other test tests for a template.